CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.28.2]
```

Comment:
I think because this is feature addition, this should be minor release instead of patch. So I propose to bump to `v1.29.0` instead. Please also update the version in `package.json` and `version.json`